.github/workflows/node-test.yml

```diff
@@ -38,6 +38,36 @@ jobs:
       - run: npm i -g npm@9.5
       - run: npm ci
       - run: npm run lint:changed-files
+      - run: npm run lint
+        working-directory: firebase-vscode
+
+  vscode_unit:
+    # Using windows to bypass an error thrown by VScode
+    # when run in an environment that does not have a screen.
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        node-version:
+          - "18"
+          - "20"
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: npm
+          cache-dependency-path: npm-shrinkwrap.json
```

Comment:
Should this be firebase-vscode/[package-lock.json instead (or in addition)? I assume we want to cache the vscode deps here.